resources/archive.rb

```diff
@@ -1,13 +1,53 @@
-unified_mode false
-actions :extract, :delete
+unified_mode true
+
 default_action :extract
 
-attribute :repo, kind_of: String, name_attribute: true
-attribute :version, kind_of: String, default: 'master'
-attribute :github_user, kind_of: String
-attribute :github_token, kind_of: String
-attribute :host, kind_of: String
-attribute :owner, kind_of: String
-attribute :group, kind_of: String
-attribute :extract_to, kind_of: String, required: true
-attribute :force, kind_of: [TrueClass, FalseClass], default: false
+property :repo, kind_of: String, name_property: true
```

Comment:
`kind_of: ` can be removed from all of these properties:
```suggestion
property :repo, String, name_property: true
```

Reply:
@tas50 I feel like Cookstyle should pick this up?

Reply:
What does `cookstyle -A` instead of `cookstyle -a` do here?

Reply:
@damacus nothing (even using the latest on the main branch of cookstyle)